update-tests.sh

```diff
@@ -5,6 +5,7 @@ set +o errexit
 
 for test in $(find tests -name '*.ts'); do
   node src/index.js $test > $test.out 2>&1
+  rc=${PIPESTATUS[0]}; if [[ $rc != 0 ]]; then exit $rc; fi
```

Comment:
What's going on here? If there are errors, the checker should fail. But sometimes that's expected. So it shouldn't be a test failure.